diff --git a/src/libopensc/card-akis.c b/src/libopensc/card-akis.c
index 9defca7f2..ccfbe7475 100644
--- a/src/libopensc/card-akis.c
+++ b/src/libopensc/card-akis.c
@@ -45,6 +45,7 @@ static struct sc_card_driver akis_drv = {
 
 static const struct sc_atr_table akis_atrs[] = {
 	{ "3b:ba:11:00:81:31:fe:4d:55:45:4b:41:45:20:56:31:2e:30:ae", NULL, NULL, SC_CARD_TYPE_AKIS_GENERIC, 0, NULL },
+	{ "3b:9f:96:81:31:fe:45:80:67:55:45:4b:41:45:12:92:31:80:73:b3:a1:80:6a", NULL, NULL, SC_CARD_TYPE_AKIS_GENERIC, 0, NULL },
 	{ NULL, NULL, NULL, 0, 0, NULL }
 };
 
